### PR TITLE
Add example of code in pre

### LINKED ISSUE #34056: <pre> code examples should show wrapping other elements like code and samp
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre

### What specific section or headline is this issue about?

Examples

### What information was incorrect, unhelpful, or incomplete?

- Two of the [examples](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre#examples) are both code samples, neither of them demonstrating the `<code>` tag.

### What did you expect to see?

Examples changed to use `<pre><code> </code></pre>`, preferably also with a single-line `<code>no pre</code>` sample like in the [HTML Standard's code example](https://html.spec.whatwg.org/multipage/grouping-content.html#the-pre-element) to emphasise when `<pre>` is necessary.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/html/element/pre`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/html/element/pre/index.md
* Last commit: https://github.com/mdn/content/commit/fdd3ac5598c3ddceb71e59949b003936ae99f647
* Document last modified: 2024-02-22T19:57:33.000Z

</details>

### PR DESCRIPTION
**Pull request by low-perry**
Updated the HTML snippets used to demonstrate the `<pre>` tags functionality. Since the snippets show code, it would be appropriate to include the `<code>` tag, as specified in the [Mdn Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code).

### Description

Added the `<code>` tag on HTML snippets that represent code.

### Additional details

Mdn docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code

### Related issues and pull requests

Fixes #34056

### CODE REVIEW

**Review by Josh-Cena — COMMENTED** [review]
The current change looks good. It would be even better to add an exmaple of using `kbd` or `samp` in `pre`, as shown in the spec. Are you interested in adding one more example? No worries if not since the issue didn't explicitly ask for it.

**Comment by low-perry** [comment]
Yeah, sure. But before I have a doubt that I need to address. 
At [Mdn Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/samp#:~:text=macOS)%20console%20session%3A-,HTML,command%20the%20user%20entered%20at%20the%20prompt%20in%20the%20sample%20text.,-CSS), if you scroll down at the examples, you can see highlighted a sample code on how to use the `<samp>` element, that includes both the `<pre>` and `<kbd>`.
What if we add a further reading / further examples and link to that page?

**Comment by Josh-Cena** [comment]
Sure, that's reasonable too 👍 You can add `samp` and `kbd` to the "See also".

**Comment by low-perry** [comment]
Thanks. 😊 I got it.

File: files/en-us/web/html/element/pre/index.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ By default, `<pre>` is a [block-level](/en-US/docs/Glossary/Block-level_content)
 
 If you have to display reserved characters such as `<`, `>`, `&`, and `"` within the `<pre>` tag, the characters must be escaped using their respective [HTML entity](/en-US/docs/Glossary/Entity).
 
+Can be used to wrap {{HTMLElement("code")}}, {{HTMLElement("samp")}}, {{HTMLElement("kbd")}}.
```
Comment (Josh-Cena) [inline files/en-us/web/html/element/pre/index.md:18]:
```suggestion
`<pre>` elements commonly contain {{HTMLElement("code")}}, {{HTMLElement("samp")}}, and {{HTMLElement("kbd")}} elements, to represent computer code, computer output, and user input, respectively.
```

**Review by Josh-Cena — APPROVED** [review]

**Pull request merged**